lib/nvmf/rdma.c

```diff
@@ -522,6 +522,14 @@ struct spdk_nvmf_rdma_transport {
 static inline void
 spdk_nvmf_rdma_start_disconnect(struct spdk_nvmf_rdma_qpair *rqpair);
 
+static void
```

Comment:
Since you added a declaration of this function, you can remove `static` linkage from the definition - according to the C standard, the later declaration of the function will have the same linkage option as the previous one. It is not critical, just a good style)